chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/xmlgenerator/SurveillanceRequirementXmlGenerator.java

```diff
@@ -26,7 +26,7 @@ public static void add(SurveillanceRequirement sr, String rootNodeName, XMLStrea
             SurveillanceNonConformityXmlGenerator.add(sr.getNonconformities(), "nonconformities", sw);
             createSimpleElement(sr.getRequirement(), "requirement", sw);
             SurveillanceResultTypeXmlGenerator.add(sr.getResult(), "result", sw);
-            SurveillanceRequirementTypeXmlGenerator.addSurveillanceRequirementType(sr.getType(), "type", sw);
+            RequirementTypeXmlGenerator.add(sr.getRequirementType(), "requirementType", sw);
```

Comment:
Just putting a note here because the XSD Validation is failing in Bamboo Maintenance Plan. The fields need to be in alphabetical order, although I feel like there are more/different errors in the XSD validation error message.